yt_dlp/extractor/_extractors.py

```diff
@@ -1704,7 +1704,7 @@
 )
 from .restudy import RestudyIE
 from .reuters import ReutersIE
-from .reverbnation import ReverbNationIE
+from .reverbnation import ReverbNationArtistIE, ReverbNationIE
```

Comment:
```suggestion
from .reverbnation import (
    ReverbNationArtistIE,
    ReverbNationIE,
)
```

Reply:
This is the default formatting, so cannot change it.

Reply:
The formatter used is not very opinionated. The version proposed will eventually produce a smaller diff though, and still pass the formatters checks.

Reply:
Figured out the difference, its the trailing comma.
Without the trailing comma, all the imports gets arranged in the same line.
With the trailing comma, each import is arranged in its own line. I've updated it for consistency.